drivers/video/video_ctrls.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_VIDEO_CTRLS_H_
```

Comment:
`ZEPHYR_INCLUDE_DRIVERS_VIDEO_VIDEO_CTRLS_H` , i believe is the correct namespace

Reply:
yes, will do.